packages/graphback-cli/src/helpers/db.ts

```diff
@@ -75,10 +75,10 @@ Run ${chalk.cyan(`npm run develop`)} to start the server.
   `)
 }
 
-export const createDB = async(): Promise<void> => {
-  checkDirectory()
+export const createDB = async (): Promise<void> => {
+  const pathForModel = checkDirectory()
```

Comment:
It will be cool if we calculate all paths in single place and pass entire object instead of individual paths.
This can be separate PR though.